playbooks/eucalyptus_basic_server.yml

```diff
@@ -0,0 +1,77 @@
+---
+# for single server community
+# This file based on `edx_sandbox.yml`
+
+- name: Configure instance(s)
+  hosts: all
+  become: True
```

Comment:
@melvinsoft Where are you seeing that? I [see it](https://github.com/appsembler/wharf/blob/develop/deploy/roles/common/tasks/main.yml#L41-L52) in the Wharf `common` role.

In Wharf, this was a workaround to allow running a task as another user. For example, I deploy as user `morgan`, but want to run a task as user `nginx`. Ansible doesn't directly support that, so we had to add e.g. `morgan ALL=(ALL) NOPASSWD: ALL` to `/etc/sudoers`.